website/docs/dependencies/0031-install-cuda-and-nvidia-drivers.md

```diff
@@ -0,0 +1,111 @@
+﻿# Install CUDA and NVIDIA GPU Drivers
+Virtual Client has a dependency component that can be added to a workload or monitor profile to install CUDA and NVIDIA drivers in Linux and Windows systems. The following section illustrates the
+details for integrating this into the profile.
+
+- [NVIDIA Official Drivers Page](https://www.nvidia.com/Download/index.aspx)
+- [CUDA Toolkit Downloads](https://developer.nvidia.com/cuda-downloads)
+
+## Supported Platform/Architectures
+* linux-x64 (Ubuntu, Debian, CentOS7, RHEL7, RHEL8, SUSE)
+* win-x64
+
+## Profile Component Parameters for Windows
+This dependency component can be used to download the drivers on Windows either from Web using Wget, or from a blob storage using the DependencyPackageInstallation. 
+The following section describes the parameters used by the individual component in the profile in Windows:
+
+| **Parameter** | **Required** | **Description**            |                 **Default**                     |
+|---------------|--------------|----------------------------|-------------------------------------------------|
+| PackageName   | Yes          | The logical name of the package that will be registered with the Virtual Client runtime. This name can be used by other profile components to reference the installation parent directory location for Drivers. |  |
+| Scenario      | No           | A name/identifier for the specific component in the profile. This is used for telemetry purposes only with components in dependency sections of the profile (i.e. cannot be used with --scenarios option on the command line). |  |
+| WinCommandLineArgs | No | The command line arguments that will be used with the Windows exe installer. | -y -s |
+| WinCudaDriversPackageDownloadedFromBlob | No | A 'true' value would mean that drivers package is downloaded from Blob and DependencyPackageInstallation is present in dependencies. A 'false' value would imply that WgetPackageInstallation is present in dependencies to download the NVIDIA drivers installer from Web | false |
```

Comment:
Is there a way we can handle this in the code? Regardless of we used wget or blobPackage download to download package, we should still be able to find the packages via package name.

Reply:
Handled in code as suggested, and removed the parameter.